forms/forms-server/src/main/java/org/bonitasoft/forms/server/api/impl/FormWorkflowAPIImpl.java

```diff
@@ -900,10 +900,9 @@ public Boolean canUserSeeHumanTask(final APISession session, final long userId,
     @Override
     public boolean canUserSeeProcessInstance(final APISession session, final long processInstanceID)
             throws ProcessInstanceNotFoundException, BPMEngineException, InvalidSessionException, UserNotFoundException, ProcessDefinitionNotFoundException {
-        return true;
-        //TODO: restore once BS-15125 has been fixed
-        //        final ProcessAPI processAPI = getBpmEngineAPIUtil().getProcessAPI(session);
-        //        boolean involvedInProcessInstance = processAPI.isInvolvedInProcessInstance(session.getUserId(), processInstanceID);
+        final ProcessAPI processAPI = getBpmEngineAPIUtil().getProcessAPI(session);
+        return processAPI.isInvolvedInProcessInstance(session.getUserId(), processInstanceID);
+        //restore this if the manager of a user should see his instances
```

Comment:
this gave me the feeling that this code was expected by engine's method which is false